source/_components/media_player.heos.page

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: "Denon HEOS speakers"
+description: "Instructions on how to integrate Denon HEOS into Home Assistant."
+date: 2019-03-06 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: heos.png
+ha_category: Media Player
+ha_release: "0.90"
+---
+
+The `heos` platform allows you to control a [Denon HEOS Speakers](http://heosbydenon.denon.com) from Home Assistant.
+
+Tested devices:
+
+- Denon HEOS-1
+
+To add a Denon HEOS speaker to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: heos
+    name: HEOS Player
+    host: IP_ADDRESS
+```
+
+{% configuration %}
+host:
+  description: "Address of the device. Example: 192.168.1.32".
+  required: false
+  type: string
+  default: Tries to resolve by UPnP (SSDP).
+name:
+  description: The name of the device
+  required: false
+  type: string
+  default: HEOS Player
+username:
+  description: The username for authentication.
+  required: false
+  type: string
+password:
+  description: The password for authentication.
+  required: false
+  type: string
+{% endconfiguration %}
+
+A few notes for platform: heos
+
+- Does not handle multiple speakers by groups.
+- Seeking is not implemented at this moment.
+
+[HEOS]: /components/media_player.heos/
```

Comment:
Are you using this link? I see no reference to it

Reply:
nope, removed